tests/integration/updateEuVehicleCategroy.intTest.ts

```diff
@@ -61,7 +61,7 @@ describe("updateEuVehicleCategory", () => {
                     expect(result.statusCode).toBe(200);
                     const techRecordWrapper: ITechRecordWrapper = JSON.parse(result.body);
                     expect(techRecordWrapper.techRecord.length).toBe(2);
-                    expect(techRecordWrapper.techRecord[0].euVehicleCategory).toBe(null);
+                    expect(techRecordWrapper.techRecord[0].euVehicleCategory).toBe(undefined);
```

Comment:
the logic checks `if(nonArchivedTechRecord[0].euVehicleCategory)`. Shouldn't you have tests for both null and undefined?

Reply:
added a test for null value as well